.ci/pytorch/test.sh

```diff
@@ -570,7 +570,7 @@ test_distributed() {
   pytest test/inductor/test_torchinductor.py -k test_multi_gpu
 
   echo "Testing distributed python tests"
-  time python test/run_test.py --distributed-tests --shard "$SHARD_NUMBER" "$NUM_TEST_SHARDS" --verbose
+  time python test/run_test.py --continue-through-error --distributed-tests --shard "$SHARD_NUMBER" "$NUM_TEST_SHARDS" --verbose
```

Comment:
```suggestion
  time python test/run_test.py --distributed-tests --shard "$SHARD_NUMBER" "$NUM_TEST_SHARDS" --verbose
```

Reply:
@malfet Yes, all additions of `--continue-through-error` are certainly *not* intended for commit, but was done in this PR as a temporary measure to catch the entire list of failures instead of failing fast and resulting in many more iterations of skips/fixes. Now that you've done an initial review and it looks good, we can take this out and let another round of CI runs finish.

Reply:
Removed `--continue-through-error` in https://github.com/pytorch/pytorch/pull/97988/commits/34795f70287defe82b5bc1198ee303a638049c18